src/services/api.js

```diff
@@ -1,22 +1,37 @@
-const BASE_URL = 'https://emir-ncnews.onrender.com/api';
+const BASE_URL = process.env.NODE_ENV === 'production' ? process.env.REACT_APP_BASE_URL : '';
```

Comment:
I am not really sure why this was changed? This isnt something that needs to be hidden as your api itself is public